src/Restyler/App.hs

```diff
@@ -80,7 +80,7 @@ instance HasGitHub StartupApp where
     runGitHub req = do
         logDebug $ "GitHub request: " <> displayShow (DisplayGitHubRequest req)
         auth <- OAuth . encodeUtf8 . oAccessToken <$> view optionsL
-        result <- appIO OtherError $ do
+        result <- liftIO $ do
```

Comment:
Since we do `dieAppError . OtherError` in the global handler, letting this just blow out as-is will be the same behavior either way. So might as well keep it simpler here.